### ISSUE TITLE
False positive "unresolved namespace" warning in data_readers.clj

### ISSUE THREAD

**Issue by rap1ds** [clj-kondo/clj-kondo#892]
**version**

clj-kondo v2020.05.09

**platform**

Native, docker and JVM

**problem**

From https://clojure.org/reference/reader#tagged_literals

> When Clojure starts, it searches for files named data_readers.clj at the root of the classpath. Each such file must contain a Clojure map of symbols, like this:

```clj
{foo/bar my.project.foo/bar
 foo/baz my.project/baz}
```

clj-kondo warns for unresolved namespaces and suggest adding a `require`.

**repro**

```
echo '{foo/bar my.project.foo/bar
 foo/baz my.project/baz}' > data_readers.clj
clj-kondo --lint data_readers.clj
data_readers.clj:1:2: warning: Unresolved namespace foo. Are you missing a require?
data_readers.clj:1:10: warning: Unresolved namespace my.project.foo. Are you missing a require?
data_readers.clj:2:10: warning: Unresolved namespace my.project. Are you missing a require?
linting took 8ms, errors: 0, warnings: 3
```

**expected behavior**

No errors/warnings about the `foo` namespace.

I'm not sure what's the expected behaviour for `my.project.foo` and `my.project` warnings. Either a) don't report them or b) check if those vars exist in the project and warn if they don't.

Or maybe just simply disable unresolved namespace linter if the file name is `data_readers.clj`?

**Comment by borkdude** [comment]
@rap1ds 

> Or maybe just simply disable unresolved namespace linter if the file name is data_readers.clj?

Yes. We do exactly this for `project.clj` too.

**Comment by jdormit** [comment]
Is there a workaround for this in the meantime, something I could put in my `.clj-kondo/config.edn`?

**Comment by jdormit** [comment]
Ah, I figured it out. Posting here for anyone else who gets stuck on this:
```edn
{:output {:exclude-files ["src/data_readers.clj"]}}
```
